src/core/index.ts

```diff
@@ -1,3 +1,3 @@
 export { setToken } from './setToken';
 export { spotifyApi } from './spotifyApi';
-export type { Token, QueryFunction, QueryConstructor } from './spotifyApi';
```

Comment:
Use named barrel exports to more clearly identify any name conflicts.

I recommend: 
Name: Indexify
Id: JordanEckowitz.indexify
Description: Generate an index file for js/ts exports in a directory
Version: 1.0.2
Publisher: Jordan Eckowitz
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=JordanEckowitz.indexify